frontend/src/metabase/databases/types.ts

```diff
@@ -1,17 +1,22 @@
 import { ComponentType, ReactNode } from "react";
 import {
+  DatabaseId,
   DatabaseSchedules,
   EngineFieldOption,
   EngineFieldType,
 } from "metabase-types/api";
 
 export interface DatabaseValues {
+  id?: DatabaseId;
   name: string;
   engine: string | undefined;
   details: Record<string, unknown>;
   schedules: DatabaseSchedules;
   auto_run_queries: boolean;
   refingerprint: boolean;
+  is_sample: boolean;
+  is_full_sync: boolean;
+  is_on_demand: boolean;
 }
```

Comment:
Should we consider extending the [`Database` type from `metabase-types/api`](https://github.com/metabase/metabase/blob/b6e50930239b9cb17c69ee289cba486581d7a442/frontend/src/metabase-types/api/database.ts#L12) and building `DatabaseValues` by stuff like `Pick<Database, ...>` ?

Reply:
Agree, I'll reconsider this in one of the future PRs. There is some duplication between `Database`, `DatabaseInfo` and `DatabaseValues` we need to solve